### PR TITLE
feat(web): add preference for thumbnail playback on hover

### PR DESCRIPTION
**Pull request by samholton**
Adds a user preference to disable video playback when hovering on the thumbnail. 

I added `playbackOnIconHover` to enable playback by hovering over the play icon even when preference is disabled. I think this is good functionality, but could still leaves a small target to accidentally start playing the video.

True is default value

Addresses https://github.com/immich-app/immich/discussions/6283

app settings

### CODE REVIEW

File: web/src/lib/components/user-settings-page/appearance-settings.svelte [hunk 1]
```diff
@@ -105,6 +105,15 @@
         />
       </div>
 
+      <div class="ml-4">
+        <SettingSwitch
+          title="Play Video Thumbnail on Hover"
```
Comment (mertalev) [inline web/src/lib/components/user-settings-page/appearance-settings.svelte:110]: Most of the titles in the user settings seem to have normal casing, the exceptions being Default Locale and now this setting. I think it's best to be consistent.

**Review by mertalev — APPROVED** [review]

**Comment by alextran1502** [comment]
Hello, thank you for the PR and the continuous contribution <3!

The only thing I am not sure about this PR is whether the option should belong to the Appearance group. I think it should be grouped as "Behavior" of some types. What do you think about this?

File: web/src/lib/components/user-settings-page/appearance-settings.svelte (hunk 1 above)
Comment (samholton) [reply web/src/lib/components/user-settings-page/appearance-settings.svelte:110]: Good point. Server settings is all uppercase. Some of user settings is all uppercase like under Account. I'll fix just my new item and do a separate PR to make the others consistent.

**Comment by samholton** [comment]
> Hello, thank you for the PR and the continuous contribution <3!
> 
> The only thing I am not sure about this PR is whether the option should belong to the Appearance group. I think it should be grouped as "Behavior" of some types. What do you think about this?

Yeah, I was trying to figure out the best place to put it. I ended up here mostly because `Display original photos` is here. I can create a new `Behavior` top level group and move `Display original photos` and `Play video thumbnail on hover`

**Comment by jrasm91** [comment]
Maybe change appearance to "App Settings"

**Comment by samholton** [comment]
> Maybe change appearance to "App Settings".
> 
> IMO all the settings that are just for the web client itself should be grouped together.

How about memories and trash? Only a single memories for now but could maybe test that later?

**Comment by samholton** [comment]
I left memories outside apps because its an actual user setting vs client side preference

**Comment by jrasm91** [comment]
> I left memories outside app settings because its an actual user setting vs client side preference

Yeah, I think client-side preferences can all be under app settings and things that are persisted to the server can have their own feature section.

**Review by jrasm91 — APPROVED** [review]

**Pull request merged**